python-sdk-example/test-requirements.txt

```
@@ -1,3 +1,3 @@
 pytest==6.2.5
-signadot-sdk==0.2.3
+signadot-sdk==0.2.5
```

It's 0.2.4 for python sdk: https://pypi.org/project/signadot-sdk/

```suggestion
signadot-sdk==0.2.4
```

---

Yeah, spotted this and then added strict error checking to the script also. Should be fixed in the latest commit.